GettingStarted.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Getting started</title>
+</head>
+<body>
+    <h2>Getting started</h2>
```

Comment:
Please wrap each parent elements on this page on `div` and add class for `div`. How I wrote in previous review